dapp/src/components/tabs.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+import styled from "styled-components"
+import { Link } from "gatsby"
+import { Pill } from "./common"
+
+type Props = {
+  proposalsCount: string
+  requestsCount: string
+  availableBalance: string
+}
+
+const Tabs = ({ proposalsCount, requestsCount, availableBalance }: Props) => {
+  return (
+    <Wrapper>
+      <LinkItem to="/proposals/" activeClassName="active">
+        Proposals <Pill size="md">{proposalsCount}</Pill>
+      </LinkItem>
+      <LinkItem to="/requests/" activeClassName="active">
+        Requests <Pill size="md">{requestsCount}</Pill>
+      </LinkItem>
+      <Rest>
+        <span>
+          Available {new Intl.NumberFormat().format(Number(availableBalance))}{" "}
```

Comment:
Here maybe we can put this calculation under an useMemo() to avoid reformatting on each rerender